src/CTA.WebForms2Blazor/ControlConverters/GridViewControlConverter.cs

```diff
@@ -21,7 +72,32 @@ public GridViewControlConverter() : base()
 
         public override HtmlNode Convert2Blazor(HtmlNode node)
         {
-            throw new NotImplementedException();
+            node.OwnerDocument.OptionOutputOriginalCase = true;
+            
+            var itemTypeAttr = node.Attributes.AttributesWithName("itemtype").FirstOrDefault();
+            bool hasItemType = itemTypeAttr != null;
+            var itemType  = itemTypeAttr?.Value ?? "PleaseReplaceWithItemTypeHere";
+
+            IEnumerable<Attribute> newItemTypeAttr = new List<Attribute>()
+            {
+                new Attribute("ItemType", itemType)
+                //"ItemType=" + itemType
+            };
+            foreach (string subControl in SubControls)
+            {
+                //Todo: make sure not adding itemtype if already exists
+                UpdateInnerHtmlNode(node, "asp:" + subControl, newName: subControl, newAttributes: newItemTypeAttr);
+            }
+
+            IEnumerable<Attribute> itemTemplateContextAttr = new List<Attribute>()
+            {
+                new Attribute("Context", "Item")
+                //"Context=Item"
+            };
+            UpdateInnerHtmlNode(node, "ItemTemplate", newAttributes: itemTemplateContextAttr);
+            
```

Comment:
Similar to above comment, assigning "ItemTemplate" to a variable called `controlToUpdate` or `elementToUpdate` would help w/ readability.